core/src/main/scala/kafka/log/LogManager.scala

```diff
@@ -327,10 +327,15 @@ class LogManager(val logDirs: Array[File],
         val needToStopCleaner: Boolean = truncateOffset < log.activeSegment.baseOffset
         if (needToStopCleaner && cleaner != null)
           cleaner.abortAndPauseCleaning(topicPartition)
-        log.truncateTo(truncateOffset)
-        if (needToStopCleaner && cleaner != null) {
-          cleaner.maybeTruncateCheckpoint(log.dir.getParentFile, topicPartition, log.activeSegment.baseOffset)
-          cleaner.resumeCleaning(topicPartition)
+        try {
+          log.truncateTo(truncateOffset)
+          if (needToStopCleaner && cleaner != null) {
```

Comment:
No need for braces if there's a single statement in the `if`. Same for the code below.